src/staff-handbook/supporting-wellbeing/getting-support-internally.md

```diff
@@ -0,0 +1,122 @@
+
+### Buddies 
+
+Once you have passed your 4 month probation, we would like you to choose a buddy within the company to support you.
+
+We recommend this person doesn’t work directly with you, someone in a different role and that you don’t work closely with on projects, as this will give you a second point of view and a whole new set of experiences to draw on.
+
+This is not mandatory, just recommended and similarly not everyone may wish to be a buddy - don’t be afraid to ask or say no. It can be reciprocal but doesn’t have to be. Also, if you feel that a change of buddy would benefit you for whatever reason or that you’re no longer able to be a buddy, this should also not be an awkward situation. If you have any concerns, have a chat with the People Manager.
+
+The buddy system is there to help us all improve, to make sure we’re doing the right kind of work and to check that we’re dealing with practical problems we’re facing. It’s not line management, so it’s not the right place to talk about holiday, salary, sickness, absence or anything disciplinary. If you need to talk about those things, talk to the People Manager.
+
+If you want to know more about how the buddy system works, how often to meet and suggested discussion topics, see the buddy crib sheet. This shouldn’t be a huge time commitment, ideally just a pleasant Friday meet up.
+
+Your delivery lead (for those on client work) #
+There will be opportunities within the delivery team to raise any issues or concerns you might have e.g. retrospectives or planning sessions. As well as this, or if you’d prefer to talk one on one, the delivery lead working on the project will be available to provide support and guidance.
+
+### Directors and Heads-of 
+
+All members of the Directors and heads-of team are here to help answer any questions you might have. Your own Director and/or Head-of profession or location is there to provide support on matters relating to dxw and your career with us, as well as profession specific questions. Similarly, they are here if you need to escalate any job-related concerns.
+
+### Our CEO, Dave 
+
+At dxw, all doors are open if you need to raise a concern or just have a chat - this includes our MD. Dave is always available to discuss any questions you have about the company and its future. Equally, if you feel you need to escalate anything to him, don’t hesitate to get in touch. Dave’s diary is open and you can see it on Google Calendar. If you would like to schedule some time with him, please feel free to book something.
+
+
+## Private Slack channels 
+
+We encourage everyone at dxw to be open about their identities and the challenges they experience at work, but sometimes we want to only talk to people whose experiences more closely resemble our own. To enable that, we have created a number of private channels:
+
+#identity-lgbtqia-plus (contact a member of @identity-lgbtqia-plus-admins to join)
```

Comment:
We should add `and #identity-lgbtqia-plus-workers` (it was created a few months ago)